apps/sync/index.html

```diff
@@ -11,4 +11,7 @@
        are still in development.
      </p>
   </body>
+  <script src="js/fxsync-webcrypto/stringconversion.js"></script>
```

Comment:
Add the MPL2 header to this file, please.

https://www.mozilla.org/en-US/MPL/headers/